src/components/field/field.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="field">
+    <div class="field-name">{{name}}</div>
+    <div class="field-value">
+      <slot/>
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: 'field',
+  props: ['name']
+}
+</script>
+<style scoped lang='scss'>
+.field {
+  display:flex;
+  justify-content:space-between;
+  padding:8px 0;
+}
+.field-name {
+  @include font-size(s);
+  color:$black;
+  opacity:0.7;
+}
+.field-value {
+  @include font-size(m);
+  font-family:'Roboto Mono', monospace;
+  align-items:center;
+  font-weight:500;
+  display:flex;
+  vertical-align:middle;
```

Comment:
How should the content of value cell be aligned? On the screenshot, it looks like `text-align: right`.

Reply:
small mistake, `align-items` has no effect

Reply:
I think you can remove `align-items`, `display:flex`, and `vertical-align:middle` from here and add `align-items: center;` to `.field`.

Reply:
ok, why?